src/dlgConnectionProfiles.cpp

```diff
@@ -864,10 +864,16 @@ QString dlgConnectionProfiles::getDescription(const QString& hostUrl, const quin
 
     } else if (hostUrl == qsl("mume.org")) {
         return qsl("Multi-Users in Middle-earth (MUME) is a highly competitive world PvP DikuMUD, set in J. R. R. Tolkien’s fictional world of Middle-earth, as described in The Hobbit and "
-                              "The Lord of the Rings, where players may choose to join the epic war between the forces of Sauron and the armies of the Free peoples. In MUME players can explore, "
-                              "role-play, acquire achievements, and complete quests across many challenging locations across Middle-earth such as Lothlórien, the Shire, Bree, Rivendell, Goblin-town,"
-                              " Mirkwood, Dol Guldur, and the Mines of Moria. The game is completely at no cost to play and has been continually enhanced since its inception in the fall of 1991.");
-
+                "The Lord of the Rings, where players may choose to join the epic war between the forces of Sauron and the armies of the Free peoples. In MUME players can explore, "
+                "role-play, acquire achievements, and complete quests across many challenging locations across Middle-earth such as Lothlórien, the Shire, Bree, Rivendell, Goblin-town,"
+                " Mirkwood, Dol Guldur, and the Mines of Moria. The game is completely at no cost to play and has been continually enhanced since its inception in the fall of 1991.");
+    } else if (hostUrl == qsl("mud.ren")) {
+        return qsl("天下风云出我辈，一入江湖岁月催。"
+                "皇图霸业谈笑中，不胜人生一场醉。"
+                "提剑跨骑挥鬼雨，白骨如山鸟惊飞。"
+                "尘事如潮人如水，只叹江湖几人回。"
+                ""
+                "中文开源武侠MUD炎黄群侠传，游戏包括25大门派和5大世家，正邪只在一念间；近千门武学等你学习，上百种任务随你体验；让自己成为一代宗师，江湖笑，恩怨了。");
```

Comment:
If you want a blank line between "paragraphs" of text you need to insert ***two*** ``\n`` - the text is autowrapped in the Qt widget that we use for displaying this text.
```suggestion
        return qsl("天下风云出我辈，一入江湖岁月催。"
                "皇图霸业谈笑中，不胜人生一场醉。"
                "提剑跨骑挥鬼雨，白骨如山鸟惊飞。"
                "尘事如潮人如水，只叹江湖几人回。\n\n"
                "中文开源武侠MUD炎黄群侠传，游戏包括25大门派和5大世家，正邪只在一念间；近千门武学等你学习，上百种任务随你体验；让自己成为一代宗师，江湖笑，恩怨了。");
```
Before:
![image](https://user-images.githubusercontent.com/6163092/178104715-c730c95e-4863-49cb-b5fb-4e6f895a434c.png)
After:
![image](https://user-images.githubusercontent.com/6163092/178104415-ab84dd9f-737a-4ed7-8f99-b760670200d5.png)